config/extenders/codeSplitting.js

```diff
@@ -72,18 +73,40 @@ export default (webpackConfig, buildOptions) => {
   if (etpIndex > -1) {
     webpackConfig.plugins.splice(etpIndex, 1);
   }
+
   // Add some plugins for css code splitting
   webpackConfig.plugins.push(
     ...removeNil([
-      ifDevClient(new WriteFilePlugin()),
-      ifClient(new ExtractCssChunks()),
-      ifClient(
+
+      ifClient(new ExtractCssChunks({ filename: '[name]-[contenthash].css' })),
+
+      ifProdClient(() => new ChunkManifestWebpackPlugin({
+        filename: '../manifest.json',
+        manifestVariable: '__WEBPACK_MANIFEST__',
+      })),
+
+      // To make sure chunk hashes stay the same if their contents don’t change
+      // see: https://webpack.js.org/guides/caching/#module-identifiers
+      ifClient(new webpack.HashedModuleIdsPlugin()),
+
+      // Add vendor code chunk
+      ifProdClient(
         new webpack.optimize.CommonsChunkPlugin({
-          names: ['bootstrap'], // needed to put webpack bootstrap code before chunks
+          name: 'vendor',
           filename: isDev ? '[name].js' : '[name]-[chunkhash].js',
-          minChunks: Infinity,
+          minChunks: module => /node_modules/.test(module.resource),
```

Comment:
What does this do?

Reply:
module.context && module.context.includes("node_modules");

Reply:
This adds every module imported from `node_modules` into a single chunk. Will add a comment and make it a little bit clearer.